web/src/containers/activity/CostAllocateFFP.js

```diff
@@ -37,23 +37,25 @@ class CostAllocateFFP extends Component {
         <div className="clearfix mxn1">
           {byYearData.map(
             ({ year, total, totalNetOther, ffpSelectVal, allocations }) => (
-              <div key={year} className="col col-12 sm-col-4 px1">
+              <div key={year} className="col col-12 sm-col-6 mb2 px1">
                 <div className="p2 bg-gray-lightest">
                   <div>{year}</div>
-                  <div className="h3 bold mono">{formatMoney(total)}</div>
+                  <div className="h3 bold mono truncate">{formatMoney(total)}</div>
                   <hr />
                   <DollarInput
+                    wrapperClass="lg-col-8 mb2"
                     name={`cost-allocate-other-${year}`}
                     label={t('activities.costAllocate.ffp.labels.other')}
                     value={costAllocation[year].other}
                     onChange={this.handleOther(year)}
                   />
                   <div>{t('activities.costAllocate.ffp.totalNetOther')}</div>
-                  <div className="h3 bold mono">
+                  <div className="h4 bold mono truncate">
```

Comment:
Question for @lauraponce @quarterback @NAretakis @jeromeleecms:

Is truncating these dollar numbers okay if they get too big?  (It looks like `$123,345,679...`)  It looks like the cost allocation component can handle into the hundreds of millions before it truncates so ***hopefully*** that should be fine.  (Goodness I hope you don't see many hundred million dollar APDs...)

Reply:
i think the solution for really big numbers just not be truncating but utilizing SI prefix (https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes) -- for example, `$123.45B` or `1.21M`